src/site/content/en/reliable/codelab-service-workers/index.md

```diff
@@ -122,9 +122,8 @@ Now that you've got the code added to `register-sw.js` and `service-worker.js`
 files, it's time to visit the Live version of your sample project, and observe
 the service worker in action.
 
-- To preview the site, mouse over the editor, press the **App** button, then the
-  **Show** button.
--  Open the DevTools by pressing `CMD + OPTION + i` / `CTRL + SHIFT + i`.
+{% Instruction 'preview' %}
+{% Instruction 'devtools' %}
 -  Click on the **Console** tab.
```

Comment:
```suggestion
{% Instruction 'devtools-console', 'ul' %}
```

Reply:
I think for these changes we'll also want to get rid of the devtools shortcode above because it's included in `devtools-console`.